config/locales/da-DK.yml

```diff
@@ -21,7 +21,7 @@ da-DK:
       sended: "En email er blevet sendt til '%{email}' med instruktioner for at nulstille dit kodeord."
       user_not_found: "Kan ikke finde en bruger med '%{email}'."
       password_not_required: "Denne bruger kræver ikke et kodeord. Log ind med '%{provider}' konto i stedet."
-      missing_passwords: "Du skal fylde alle felter ud som indeholder 'Password' og 'Password confirmation'."
+      missing_passwords: "Du skal udfylde alle felter med kodeord og gentagelse af kodeord."
```

Comment:
Perhaps better as `Du skal udfylde både kodeord og gentagelse af kodeord`? As it stands now it reads like there's multiple password and multiple confirmation fields.

Reply:
Usually `confirmation` is translated to `bekræftelse` btw.